source/_patterns/basics/forms/form-group.hbs

```diff
@@ -0,0 +1,47 @@
+{{!--
+formRightAligned - when set to true it'll display the label on the right of the form
+formHelp - when set to true it'll display a help text line
+--}}
+
+{{#if formRightAligned}}
+  <div class="form-group row {{ formGroupModifierClass }}">
+    <label class="form-control-label col-sm-2 col-form-label" for="{{formInputId}}">{{ formLabel }}</label>
+
+    <div class="col-sm-10">
+      {{#if formTextarea}}
```

Comment:
You could probably move the `{{else}}` to here and just include the `<div ...><label ...>` elements within the `{{else}}` block. Then you would probably need another `{{#if formRightAligned}}` for the closing `</div>` for `<div class="col-sm-10">`. This would let you have one place where the different form elements are defined.